Appz/Appz/Apps/Mail.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Mail.swift
+//  Appz
+//
+//  Created by Suraj Shirvankar on 12/5/15.
+//  Copyright © 2015 kitz. All rights reserved.
+//
+
+import Foundation
+
+
+public extension AvailableApplications {
+    
+    public struct Mail: ExternalApplication {
+        
+        public let scheme = "mailto:"
+        public let fallbackURL: String? = ""
+    }
+    
+    public func mail(action: Mail.Action) -> Bool {
+        return appCaller.launch(Mail(), action: action)
+    }
+}
+
+// MARK: - Actions
+
+public extension AvailableApplications.Mail {
+    
+    public enum Action {
+        case Email(email: String, subject: String, body: String)
+        case EmptyEmail(email: String)
+        case Subject(subject: String)
+    }
+}
+
+extension AvailableApplications.Mail.Action: ExternalApplicationAction {
+    
+    public var path: String {
+        
+        switch self {
+        case .Email(let email, let subject, let body):
+            return "\(email)?subject=\(subject)&body=\(body)"
```

Comment:
We probably need some sort of escapes here, see:
https://github.com/SwiftKitz/Appz/blob/master/Appz/Appz/Entity/ExternalApplication.swift#L37